pyproject.toml

```diff
@@ -131,7 +127,7 @@ osx-platform = [
 
 gtk-platform = [
     "twisted[all-non-platform]",
-    "pygobject",
+    "pygobject < 3.52.1",
```

Comment:
Just asking. Why do we require a maximum version ?

Reply:
Restricting the max version to python3.9 envs fixed the OSX issues.

Reply:
I think that the right way to express this would be an environment specifier, rather than a hard cap like this?